### ISSUE TITLE
Recurring reminders interval trouble

### ISSUE THREAD

**Issue by HarHarLinks** [anoadragon453/matrix-reminder-bot#51]
Maybe this only happens when the interval is large enough. Maybe it's due to rounding down from 59.99999 seconds.

 I don't know whether it's just the message or the actual interval, too.

Example:
- `!remindme every 2w; monday 14:00; text`
- OK, I will remind you on Oct 12 2020, 14:00, and again every 1 week, 6 days, 23 hours, 59 minutes and 59 seconds!

**Comment by Bubu** [comment]
I have another one: `!remindme every 1 month; 20th of February; text` --> OK, I will remind you on Feb 20 2021, 00:00, and again every 4 weeks!

Hey, that's not how months work ;-p.

**Comment by anoadragon453** [comment]
@Bubu Ah, yes. I'm afraid that's not how `every`-based recurring reminders work as well :D They work based off a static interval. A cron reminder would be better suited for what you're trying to do.

**Comment by Bubu** [comment]
I did go with a cron reminder, yes.

And I see now as how the `every` thing would be fundamentally incompatible with thisright now... it just felt like the natural thing to try :D

**Comment by anoadragon453** [comment]
Indeed, there's probably some documentation that could be added here to fix this, but I don't want to make `!help` too cluttered. 

Perhaps if people try to use `month` in the entry field the bot could warn them that not all months are created equal.

**Comment by HarHarLinks** [comment]
I can't reproduce my original issue anymore, the interval is now `1209600`.

**Comment by HarHarLinks** [comment]
i believe the remaining issue is #45
